lighthouse-core/audits/accessibility/button-name.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * @fileoverview Ensures buttons have discernible text.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit');
+
+class ButtonName extends AxeAudit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Accessibility',
+      name: 'button-name',
+      description: 'buttons have an accessible name.',
```

Comment:
spotted this one. The lower case looks weird: 

![screen shot 2017-03-22 at 11 34 08 am](https://cloud.githubusercontent.com/assets/238208/24214705/8d9b07e4-0ef3-11e7-8291-7e364c75bc27.png)


How about buttons -> \`&lt;button>\`s